### PR TITLE
[Docs] Adding hyperlinks for Isovalent Enterprise Features

### PR DESCRIPTION
**Pull request by amitmavgupta**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/clustermgmt/networking/networking-and-security.md [hunk 1]
```diff
@@ -126,6 +126,12 @@ For next steps on leveraging Network Policy, we encourage you to explore:
 
 ## Additional Cilium Features
 
-Many advanced features of Cilium are not yet enabled as part of EKS Anywhere, including:  Hubble observability, DNS-aware and HTTP-Aware Network Policy, Multi-cluster Routing, Transparent Encryption, and Advanced Load-balancing.
+Many advanced features of Cilium are not yet enabled as part of EKS Anywhere, including:
```
Comment (chrisdoherty4) [inline docs/content/en/docs/clustermgmt/networking/networking-and-security.md:129]:
```suggestion
Some advanced features of Cilium are not enabled as part of EKS Anywhere, including:
```

File: docs/content/en/docs/clustermgmt/networking/networking-and-security.md [hunk 2]
```diff
@@ -126,6 +126,12 @@ For next steps on leveraging Network Policy, we encourage you to explore:
...
 ## Additional Cilium Features
 
-Many advanced features of Cilium are not yet enabled as part of EKS Anywhere, including:  Hubble observability, DNS-aware and HTTP-Aware Network Policy, Multi-cluster Routing, Transparent Encryption, and Advanced Load-balancing.
+Some advanced features of Cilium are not enabled as part of EKS Anywhere, including:
+
+* Hubble observability (https://docs.isovalent.com/operations-guide/features/network-visibility/index.html)
+* DNS-aware and HTTP-Aware Network Policy (https://docs.isovalent.com/quick-start/policy_lifecycle.html)
+* Multi-cluster Routing (https://docs.isovalent.com/operations-guide/features/cluster-mesh/index.html)
+* Transparent Encryption (https://docs.cilium.io/en/v1.13/security/network/encryption/)
+* Advanced Load-balancing (https://docs.isovalent.com/operations-guide/features/cilium-standalone-gateway.html)
```
Comment (chrisdoherty4) [inline docs/content/en/docs/clustermgmt/networking/networking-and-security.md:135]:
```suggestion
* [Hubble observability](https://docs.isovalent.com/operations-guide/features/network-visibility/index.html)
* [DNS-aware and HTTP-Aware Network Policy](https://docs.isovalent.com/quick-start/policy_lifecycle.html)
* [Multi-cluster Routing](https://docs.isovalent.com/operations-guide/features/cluster-mesh/index.html)
* [Transparent Encryption](https://docs.cilium.io/en/v1.13/security/network/encryption/)
* [Advanced Load-balancing](https://docs.isovalent.com/operations-guide/features/cilium-standalone-gateway.html)
```

**Comment by chrisdoherty4** [comment]
Sorry for the delay. Final suggestion and we're good to go.

**Review by chrisnegus — COMMENTED** [review]
I have no objections, but I'd like @csplinter to give his okay.

**Comment by csplinter** [comment]
lgtm

**Review by drewvanstone — APPROVED** [review]

**Review by csplinter — APPROVED** [review]

**Pull request merged**